server.js

```diff
@@ -2,19 +2,26 @@ import express from 'express';
 import cors from 'cors';
 import mongoose from 'mongoose';
 import 'dotenv/config';
-import router from './api/index.js';
+import './config/config-passport.js';
+import usersRouter from './routes/users.routes.js';
+import contactsRouter from './routes/contacts.routes.js';
 
 const app = express();
 
 app.use(express.json());
 app.use(cors());
-app.use('/api', router);
+app.use('/api', usersRouter);
+app.use('/api', contactsRouter);
```

Comment:
Tu możesz dodać prefix endpointów ('/api/contacts', 'api/users'). Pozbędziesz się pisania 5 razy tego w routes i łatwiejsze będzie to w edycji czy debugowaniu (łatwiej sprawdzić jedno miejsce niż 10)

Reply:
corrected